test/tests.js

```diff
@@ -0,0 +1,14 @@
+import isYes from '../isYes.js';
+const test = QUnit.test;
+
+test('isYes', function(assert) {
+    //Arrange
```

Comment:
Please set up parameters for your tests, otherwise it's very difficult to tell exactly what you're testing.